threescale_api/auth.py

```diff
@@ -19,7 +20,7 @@ def __call__(self, request):
 
         if self.location == "authorization":
             credentials = credentials.values()
-            auth = requests.auth.HTTPBasicAuth(*credentials)
+            auth = requests.auth.HTTPBasicAuth(credentials["username"], credentials["password"])
```

Comment:
unlikely correct because of `credentials = credentials.values()` (not a dict anymore)

Reply:
Pylint complained about `auth = requests.auth.HTTPBasicAuth(*credentials)` and I've missed `credentials = credentials.values()`, I'll fix this.